test/Microsoft.Azure.Devices.Edge.Test/helpers/TestResultCoordinatorUtil.cs

```diff
@@ -86,8 +86,16 @@ public static async Task ValidateResultsAsync()
             HttpClient client = new HttpClient();
             HttpResponseMessage response = await client.GetAsync(TestResultCoordinatorUrl);
             var jsonstring = await response.Content.ReadAsStringAsync();
+            bool isPassed;
+            try
+            {
+                isPassed = (bool)JArray.Parse(jsonstring)[0]["IsPassed"];
+            }
+            catch
```

Comment:
might as well be `finally`.

Reply:
No, it needs to be a catch. If the parse is successful it shouldn't be set to false